src/api/apiData.js

```diff
@@ -22,6 +27,7 @@ const getApiDataSelector = () =>
     ],
     (
       /*number*/ tableHeight,
+      /*number*/ bodyAndHeaderHeight,
```

Comment:
I would suggest lets avoid creating a new return value (bodyAndHeaderHeight) rather update the `tableHeight` value only, this way the tableHeight value would be correct in both the cases (maxHeight and height) and ResizeCell could still rely on tableHeight only. Then we would also wont need to make any change in ResizeCell.js file.

Reply:
ok. I changed this to use the already existing tableHeight